rest/route/stats.go

```diff
@@ -328,7 +281,84 @@ func (tsh *testStatsHandler) readStartAt(startAtValue string) (*stats.StartAt, e
 	}, nil
 }
 
-// makeStartAtKey creates a key string that can be used as a start_at value to fetch
+///////////////////////////////////////////////
+// /projects/<project_id>/test_stats handler //
+///////////////////////////////////////////////
+
+type testStatsHandler struct {
+	sc data.Connector
+	statsHandler
+}
+
+func (tsh *testStatsHandler) Factory() gimlet.RouteHandler {
+	return &testStatsHandler{sc: tsh.sc}
+}
+
+func (tsh *testStatsHandler) Parse(ctx context.Context, r *http.Request) error {
+	vals := r.URL.Query()
+	project := gimlet.GetVars(r)["project_id"]
+	tsh.filter = stats.StatsFilter{Project: project}
+
+	err := tsh.statsHandler.parseStatsFilter(vals)
+	if err != nil {
+		return err
```

Comment:
The error is a gimlet.ErrorResponse that is created by parseStatsFilter() with the idea that it could be returned directly so I didn't feel it needed to be wrapped.

Reply:
gimlet components unwrap errors, so you don't have to worry about that.

Reply:
Ok, done.